src/crs/projected.jl

```diff
@@ -89,3 +89,6 @@ end
 
 # avoid converting coordinates with the same type as the first argument
 Base.convert(::Type{C}, coords::C) where {Datum,C<:Projected{Datum}} = coords
+
+Base.convert(::Type{Cartesian}, coords::Projected) = convert(Cartesian, convert(LatLon, coords))
+Base.convert(P::Type{<:Projected}, coords::Cartesian) = convert(P, convert(LatLon, coords))
```

Comment:
Use the letter `C` for `CRS` types and change first method to accept `Cartesian` and `Cartesian{Datum}`:
```suggestion
Base.convert(C::Type{<:Cartesian}, coords::Projected) = convert(C, convert(LatLon, coords))
Base.convert(C::Type{<:Projected}, coords::Cartesian) = convert(C, convert(LatLon, coords))

```